routes/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const users = require('./users');
+const validatePermisions = require('../middleware/validatePermissions');
+const { jwtVerify } = require('../handlers');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  const jwt = req.cookies.token;
+  const isValid = jwtVerify(jwt);
+  const userData = {};
+  if (isValid) {
```

Comment:
You can actually conditionally add properties to the `userData` object here to shorten up this code. It's not necessary, of course, but it would look like so:

```
res.render( 'index', { layout: 'default', { ...isValid && { userData.role: isValid.role, userData.usr: isValid.role } };
```